day2/01_Workflows/02_Centralized.md

```diff
@@ -71,6 +71,28 @@ for integrating and sharing code changes between repositories.
 
 ****
 
+~~~ENDSECTION~~~
+
+!SLIDE smbullets
+# Centralized Workflow - Merge Conflicts
+
+* Conflicts arise when the same lines in a file have changed
+* When merging Git will try to automatically integrate new changes
+* Git cannot automatically determine what is correct
+* `>>>` marks conflicts and show the differences
```

Comment:
* `===` is the barrier between the two states

Hint: those markers need to be deleted from the files

Reply:
Good idea

Reply:
Done